php/ubiquity/Dockerfile-u

```diff
@@ -0,0 +1,57 @@
+FROM php:7.3-fpm
+
+RUN apt-get -qq update
+RUN apt-get -qy install git zlib1g-dev libzip-dev nginx  nginx 
+RUN docker-php-ext-install zip opcache
+
+WORKDIR /usr/src/app
+
+
+COPY . ./
+
+RUN curl -sSL https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+RUN composer install --no-dev --prefer-dist --classmap-authoritative
+RUN composer dumpautoload -o
+
+
+  RUN composer update --dev
+  RUN chmod +x vendor/bin/Ubiquity
+  RUN ./vendor/bin/Ubiquity bootstrap dev
```

Comment:
why use `dev` ?

Reply:
I saw that the `composer update` was in `dev` by default
so the `--dev` is useless.
But other than that, the docker makes a composer install --no-dev
However, I have to install `Ubiquity-devtools` to generate the cache.

Reply:
Perhaps, `Ubiquity-devtools` in **production** setup. Indeed, cache generation is accurate in **production**

Reply:
Once in production, `devtools` are no longer useful and unnecessarily overload the composer autoloading.